com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/StackTraceRequestHandler.java

```diff
@@ -152,4 +154,21 @@ public void handle(Command command, Arguments arguments, Response response, IDeb
             }
         }
     }
+
+    private String formatMethodName(Method method, boolean showContextClass, boolean showParameter) {
+        String formattedName = method.name();
```

Comment:
Better to use StringBuilder in this method for lots of String concat

Reply:
why not use typeformatter

Reply:
fixed.